.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8"
+  - "10"
```

Comment:
Any specific reason why the version has to change?

Reply:
Version 8 does not support the regex required for `Phase2DescriptionTemplate` and `Phase3DescriptionTemplate` causing Travis to throw an error when running tests